### PR TITLE
Fix documentation on minikube installation ID docs

### PR DESCRIPTION
**Pull request by WilliamChang80**
Changed title and content for "Menginstal kubectl" on each tab for consistency, also fix id on link kubectl installation

Page: https://kubernetes.io/id/docs/tasks/tools/

### CODE REVIEW

**Comment by WilliamChang80** [comment]
I signed it

**Comment by WilliamChang80** [comment]
I signed it

**Comment by WilliamChang80** [comment]
I signed it

File: content/id/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ Hyper-V Requirements:     A hypervisor has been detected. Features required for
 {{< tabs name="tab_with_md" >}}
 {{% tab name="Linux" %}}
 
-### Menginstal kubectl
+### Instalasi kubectl
 
-Pastikan kamu mempunyai kubectl yang terinstal. Kamu bisa menginstal kubectl dengan mengikuti instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/id/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux).
+Pastikan kamu mempunyai kubectl yang terinstal. Kamu bisa menginstal kubectl dengan mengikuti instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/id/docs/tasks/tools/install-kubectl/#menginstal-kubectl-pada-linux).
```
Comment (irvifa) [inline content/id/docs/tasks/tools/install-minikube.md:68]:
```suggestion
Pastikan kamu sudah menginstal kubectl. Kamu bisa menginstal kubectl dengan mengikuti instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/id/docs/tasks/tools/install-kubectl/#menginstal-kubectl-pada-linux).
```

**Review by irvifa — CHANGES_REQUESTED** [review]
Hi 👋 thank you for your contribution I have a nitty gritty comment.

File: content/id/docs/tasks/tools/install-minikube.md (hunk 1 above)
Comment (WilliamChang80) [reply content/id/docs/tasks/tools/install-minikube.md]: I've changed it. You can check it again :D

**Comment by irvifa** [comment]
Superb 😎
/lgtm
/approve

**Pull request merged**